test/FastTests/Blittable/BlittableValidationTest.cs

```diff
@@ -0,0 +1,628 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Raven.Imports.Newtonsoft.Json;
+using Sparrow.Json;
+using Xunit;
+using Raven.Json.Linq;
+using Sparrow.Compression;
+using Sparrow.Json.Parsing;
+
+namespace FastTests.Blittable
+{
+    public class BlittableValidationTest : RavenTestBase
+    {
+        public int Size = 0xbc;
+        public byte[] SimpleValidBlittable = new byte[190]
```

Comment:
This is pretty bad, because it doesn't tell us the bittable is.
It is better to host the constant as json string, convert to blittable array, and move from there.